### ISSUE TITLE
Подборки по целям  - содержание выдач [dev]

### ISSUE THREAD

**Issue by kbochenina** [avatar-project/public-issues#64]
Ситуация: подборка по цели Стартапы

кейсы:
- выдача с другой целью (досуг), то есть в подборке по цели выдачи с другой целью - Конкурс видеороликов в рамках X конгресса молодых ученых
- выдача вообще без индикатора цели (Data Science fwdays)
- выдача с 3 другими целями (обучение, карьера, коммуникации) - ITMOTECH

Я бы ожидала что все выдачи в подборке цели будут маркированы этой целью (+ возможно другими). А то получается зашел прокачать одну цель, а прокачиваешь другие (в прогрессе)

**Comment by davidvirabian** [comment]
Да, я тоже это заметил.
На мой взгляд, это не совсем критично, так как выдачи могут косвенно относиться к цели подборки. Контекстуальное совпадение будет, благодаря чему пользователь маловероятно отреагирует негативно. Другой вопрос, что баллы будут прибавляться к другим целям.

Я бы оставил этот вопрос на пострелизный период, т.к. тут надо комплексно работать механизм "маркирования выдач по целям (на основе чего будет определяться цель? И каким образом?)

**А если искать решение "здесь и сейчас", то можно априори маркировать выдачи в подборках по соответствующей цели. 
Это решит вопрос на короткий промежуток времени. Ну а дальше можно будет уже поработать над улучшением механизма определения цели выдачи**

**Comment by danwhale** [comment]
Дичайший мисскомьюникейт. @davidvirabian Давид, мы с тобой обговаривали что, по целям мы делаем 8 подборок, где тупо совпадает иконка, верно?) 

А то, что ты там с тегами рисовал будем присылать в уведомлениях по каждой цели (ну и там да, уже возможны наслоения по разным целям, но этого будет только интереснее пользователю) 

поправь меня плз, если я не прав.

**Comment by davidvirabian** [comment]
Нет, все верно 
Мы с тобой это и обсуждали

**Comment by kbochenina** [comment]
@danwhale я вот и думала что по целевым подборкам тупо совпадает иконка, но когда я смотрела это было не так, что и вызвало к жизни данный тикет.

В итоге пока я поняла так:
- целевые мотиваторы содержат _только_ выдачи, маркированные целью мотиватора (хотя бы одна из набора целей)
- уведомление № 8 присылает другие подборки, которые на основе каких-то тэгов

@danwhale @davidvirabian подтвердите или опишите как правильно

**Comment by danwhale** [comment]
Да, я тоже понимаю так, сейчас сделано немного иначе (из за беспорядка), давайте утвердим ;)

**Comment by kbochenina** [comment]
ОК. Тикет пусть висит, @davidvirabian закрываем только если выполняется при проверке условие

- целевые мотиваторы содержат только выдачи, маркированные целью мотиватора (хотя бы одна из набора целей)

**Comment by davidvirabian** [comment]
Окей, беру под контроль – проверю по факту реализации и закрою

**Comment by hainamantik** [comment]
Просьба проверить - контент теперь подбирается по логике, описанной выше

**Comment by kbochenina** [comment]
Значки да, вижу на целевых подборках, другой вопрос что разметка не идеальна (фотки из Nat Geo, гриль-бар  и театральные новости в разделе Спорт), но тут уже остается только развести руками

**Comment by kbochenina** [comment]
Эээ нет. См в подборке по цели Творчество
![photo_2021-02-19_17-58-57](https://user-images.githubusercontent.com/5405987/108520865-2f7c9f00-72dc-11eb-8ffd-6a367c80dcd2.jpg)

**Comment by danwhale** [comment]
@kapc3d я вчера вечером сдавал баг с тем, что подборки путаются. Очень возможно, что это он.

**Comment by hainamantik** [comment]
Баг на фронте, показывается контент из другой подборки. В базе ассет не принадлежит этой подборке

**Comment by hainamantik** [comment]
@kapc3d призываю тебя, в куче тикетов уже всплыла эта проблема

**Comment by kbochenina** [comment]
@kapc3d да, я тоже призываю! даже по моему в пользовательских видела что подставляется по смыслу из другой подборки. Прямо сейчас на dev у меня в карьере подборка про искусство

**Comment by kapc3d** [comment]
приянли в работу, разбираемся
